glustercli/cmd/volume-statedump.go

```diff
@@ -0,0 +1,86 @@
+package cmd
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/gluster/glusterd2/pkg/api"
+
+	"github.com/spf13/cobra"
+)
+
+var (
+	flagStatedumpClientHost string
+	flagStatedumpVolumeName string
+	flagStatedumpClientPid  int
+)
+
+var volumeStatedumpCmd = &cobra.Command{
+	Use:   "statedump --volname <volname>",
```

Comment:
Doing it this way changes the well know volume command syntax of `gluster volume <sub-command> <volname> <other args or options>`.
The statedump command is still a volume subcommand, and should follow the others.